toolkit/scripts/toolchain/cross/binutils_cross.spec

```
@@ -0,0 +1,225 @@
+# Globals which should be in a macro file.
+# These should be set programatically in the future.
+%global _host_arch      x86_64
+%global _target_arch    aarch64
+%global _tuple          %{_target_arch}-linux-gnu
+%global _cross_name     %{_target_arch}-linux-glibc
```

```suggestion
%global _cross_name     %{_target_arch}-linux-gnu
```
Recommend changing this to gnu. Most cross compile packages use "gnu" here which implies "glibc" as their libc.

---

done